packages/fast-components-react-base/src/text-field/text-field.props.ts

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { IManagedClasses, ITextFieldClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+export enum TextFieldType {
+   date = "date",
+   email = "email",
+   hidden = "hidden",
+   month = "month",
+   number = "number",
```

Comment:
We usually have a separate component for number field, so I think this should be omitted... again thoughts?

Reply:
@nalogiudice : If you're saying that text-field should be one word, then should that also apply to the file name?  Also, I'm not understanding what you mean by Number?  Do you have an example or file we can look at?

Reply:
With unhandled props I think we can apply all functionality we need to to ensure both text and number fields are supported with a single component. I know previously we did that, although I don't recall the exact reason. Is there a specific scenario you can think of that would require two different components here?

Reply:
I think we may be able to get by with unhandled props. Perhaps an example that tests that would be beneficial in this case. I.E. uses min, max, step etc... to verify it works as expected.

@awentzel if you look on fluentweb you'll see we have a number field component and a text field component, as separate components. But, @chrisdholt 's point is valid I believe, with my caveat above.

Text field is two words, no hyphen